### ISSUE TITLE
Error with n_jobs Parameter in Parallel Execution

### ISSUE THREAD

**Issue by Ophelia-Wang** [alceballosa/deform-aneurysm-detection#2]
I encountered the following error while running the code:
ValueError: n_jobs == 0 in Parallel has no meaning

The specific error is shown in the attached screenshot.

Could you clarify how I should modify the executor(tasks) calls in src/preprocess/resample_scans.py and src/preprocess/crop_scans.py?

Should I use something like executor(tasks, n_jobs=4) by setting n_jobs to a positive integer, such as 4 or another value?

Thank you in advance for your help!
![error1_n_jobs](https://github.com/user-attachments/assets/0c63ef0d-cfcb-432e-9033-7b2fd541e068)

**Comment by alceballosa** [comment]
Hi! I think this error is happening because the path to the folder with the scans is not properly set up -thus it detects 0 scans-. Can you share the path to the folder with your scans? Also, are they nii.gz files?

**Comment by Ophelia-Wang** [comment]
@alceballosa 
Hi Alberto,  

You’re absolutely right—it was indeed a path issue.  

Actually, I loaded the `vessel_seg_og.tar` image on Windows (via Docker Desktop) and used the Docker run command to execute the shell script in the VSCode terminal. I also tried converting `run_preproc_pipeline_inf.sh` into a PowerShell script and ran it successfully.  

Thank you for your prompt response. Wish you a great day!  

Ophelia
